spec/controllers/cases_controller/unflag_for_clearance_spec.rb

```diff
@@ -19,10 +19,9 @@
       end
 
       it 'authorises' do
-        expect {
-          patch :unflag_taken_on_case_for_clearance, params: params
-        } .to require_permission(:can_unflag_for_clearance?)
-                .with_args(disclosure_specialist, flagged_case)
+        expect_any_instance_of(Case::FOI::StandardPolicy).to receive(:can_unflag_for_clearance?)
```

Comment:
Changing the way this is tested reduces the test coverage ... because we're no longer testing that it authorises the right thing. Can't we test it with `require_permission`?

Reply:
The problem is that the policy gets called twice - once by the controller, using positional parameters, and once by the state machine using named parameters.  I think we should have a video chat this afternoon to talk about how best to go about this.

Reply:
kk